api/machine/machine.proto

```diff
@@ -491,6 +493,8 @@ message FileInfo {
   uint32 uid = 10;
   // Owner gid
   uint32 gid = 11;
+  // Label is filled with the value of `security.selinux` xattr if present and requested
+  string label = 12;
```

Comment:
I think on API level (as it's hard to change later), we should return all labels.

The CLI might report a single label for now

Reply:
> all labels

You mean all xattrs?

Reply:
yes, all xattrs